src/fluree/db/time_travel.cljc

```diff
@@ -155,18 +155,13 @@
           flakes (some-> db
                          dbproto/-rootdb
                          (query-range/index-range :post
-                                                  > [const/$_commit:time 0]
-                                                  < [const/$_commit:time epoch-datetime])
+                                                  >= [const/$_commit:time epoch-datetime]
+                                                  <  [const/$_commit:time (System/currentTimeMillis)])
```

Comment:
CLJ-only, I think we have a CLJC version in `util`.

Reply:
Resolved in [e122a09](https://github.com/fluree/db/pull/273/commits/e122a09cb0ccdcd5243b6dead90a70fdf5252827) but I'll see if there's a better version in `util`.